.github/workflows/static_analysis.yaml

```diff
@@ -123,33 +121,60 @@ jobs:
           
           python3 -mvenv venv
           venv/bin/pip3 install -r requirements.txt
+          
+      - name: "Get Current CMake version"
+        id: cmake_version
+        run: |
+          work_dir=$(pwd)
+          cd ..
+          
+          cd cmake_min_version
           venv/bin/python3 cmake_downloader.py --first_minor
           output=$(venv/bin/python3 cmake_min_version.py $work_dir)
-          echo "$output"
+          echo $output
           
           # Retrieve CMake minimal version from the last line of the tool output.
```

Comment:
Please include some sample output so we can understand what it looks like.
Like that picture you sent me..

Reply:
You mean include comment in the code or print a sample to a console?

Reply:
Sample added to the code (It will be printed to console)